eclair-core/src/main/scala/fr/acinq/eclair/api/FormParamExtractors.scala

```diff
@@ -57,4 +57,10 @@ object FormParamExtractors {
     Timeout(str.toInt.seconds)
   }
 
+  implicit val pubkeyListUnmarshaller: Unmarshaller[String, List[PublicKey]] = Unmarshaller.strict { str =>
+    serialization.read[List[String]](str).map { el =>
```

Comment:
Does that parse URL-encoded json?

Reply:
There is no URL-encoding here, this only parses the form data parameter (as json array of strings) into a list of public keys.